frontend_tests/node_tests/recent_senders.js

```diff
@@ -82,19 +84,19 @@ run_test('process_message_for_senders', () => {
     // Same stream, but different topics
     const message6 = {
         stream_id: stream3,
-        id: _.uniqueId(),
+        id: next_id += 1,
```

Comment:
I'd be happy to eliminate `uniqueId` in all the places we use it in tests.  It basically just does what you're doing here (increment a global), but I prefer the more explicit style, as the semantics of `uniqueId` aren't obvious from the docs.

Here's the underscore code:

~~~
  var idCounter = 0;
  _.uniqueId = function(prefix) {
    var id = ++idCounter + '';
    return prefix ? prefix + id : id;
  };

Reply:
I went and eliminated all the other uses of `_.uniqueId` that wanted numbers.

Reply:
Thanks!